registry/rocketpool/withdraw-rocketpool.json

```diff
@@ -0,0 +1,314 @@
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [],
+          "stateMutability": "payable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "receiveVaultWithdrawalETH",
+          "inputs": [],
+          "outputs": [],
+          "stateMutability": "payable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "recycleDissolvedDeposit",
+          "inputs": [],
+          "outputs": [],
+          "stateMutability": "payable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "recycleExcessCollateral",
+          "inputs": [],
+          "outputs": [],
+          "stateMutability": "payable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "recycleLiquidatedStake",
+          "inputs": [],
+          "outputs": [],
+          "stateMutability": "payable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "version",
+          "inputs": [],
+          "outputs": [
+            {
+              "name": "",
+              "type": "uint8",
+              "internalType": "uint8",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "stateMutability": "view",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "withdrawExcessBalance",
+          "inputs": [
+            {
+              "name": "_amount",
+              "type": "uint256",
+              "internalType": "uint256",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        }
+      ],
+      "addressMatcher": null,
+      "factory": null
+    }
+  },
+  "metadata": {
+    "owner": "Rocket Pool",
+    "info": {
+      "legalName": "Rocket Pool",
+      "url": "http://rocketpool.net/"
+    }
+  },
+  "display": {
+    "formats": {
+      "withdrawExcessBalance(uint256)": {
+        "$id": null,
+        "intent": "withdrawExcessBalance(uint256)",
```

Comment:
Here you should just write the intent in the function like "withdraw Excess Balance" in a human readable sentence